src/uikit.cs

```diff
@@ -18433,4 +18433,45 @@ interface UIFontMetrics {
 #endif // !WATCH
 	}
 
+#if !WATCH
```

Comment:
Minor but could we not have avoided that with empty skeletons like we usually do?

Reply:
yes, but the errors were on an existing type and I did not want to PR to touch unrelated code.